tests/browser_e2e_tests/create_and_query_database.rs

```diff
@@ -0,0 +1,223 @@
+    let create_table_query = "CREATE TABLE test_table(fname varchar, lname varchar);";
+
+    page.fill_builder("textarea[name='query']", create_table_query)
+        .timeout(1000.0)
+        .fill()
+        .await?;
+
+    // Screenshot with create table query
+    BrowserHelpers::screenshot_compare(&page, "create_table_query", &[]).await?;
+
+    // Run the create table query
+    page.click_builder("button:has-text('Run query')")
+        .timeout(5000.0)
+        .click()
+        .await?;
+
+    // Wait for query to execute
+    tokio::time::sleep(std::time::Duration::from_millis(1000)).await;
+
+    // Screenshot after table creation
+    BrowserHelpers::screenshot_compare(&page, "table_created", &[]).await?;
+
+    // Step 7: Insert data (same as e2e tests)
+    let insert_query1 =
+        "INSERT INTO test_table (fname, lname) VALUES (\"the first\", \"the last\");";
+
+    // Clear previous query and enter insert query
+    page.fill_builder("textarea[name='query']", "")
+        .timeout(1000.0)
+        .fill()
+        .await?;
+
+    page.fill_builder("textarea[name='query']", insert_query1)
+        .timeout(1000.0)
+        .fill()
+        .await?;
+
+    page.click_builder("button:has-text('Run query')")
+        .timeout(5000.0)
+        .click()
+        .await?;
+
+    tokio::time::sleep(std::time::Duration::from_millis(1000)).await;
+
+    // Insert second row
+    let insert_query2 =
+        "INSERT INTO test_table (fname, lname) VALUES (\"the first2\", \"the last2\");";
+
+    page.fill_builder("textarea[name='query']", "")
+        .timeout(1000.0)
+        .fill()
+        .await?;
+
+    page.fill_builder("textarea[name='query']", insert_query2)
+        .timeout(1000.0)
+        .fill()
+        .await?;
+
+    page.click_builder("button:has-text('Run query')")
+        .timeout(5000.0)
+        .click()
+        .await?;
+
+    tokio::time::sleep(std::time::Duration::from_millis(1000)).await;
+
+    // Screenshot after data insertion
+    BrowserHelpers::screenshot_compare(&page, "data_inserted", &[]).await?;
+
+    // Step 8: Query the data (same SELECT as e2e tests)
+    let select_query = "SELECT * FROM test_table;";
+
+    page.fill_builder("textarea[name='query']", "")
+        .timeout(1000.0)
+        .fill()
+        .await?;
+
+    page.fill_builder("textarea[name='query']", select_query)
+        .timeout(1000.0)
+        .fill()
+        .await?;
+
+    // Screenshot with select query
+    BrowserHelpers::screenshot_compare(&page, "select_query", &[]).await?;
+
+    page.click_builder("button:has-text('Run query')")
+        .timeout(5000.0)
+        .click()
+        .await?;
+
+    // Wait for results to load
+    tokio::time::sleep(std::time::Duration::from_millis(1500)).await;
+
+    // Screenshot of query results table
+    BrowserHelpers::screenshot_compare(&page, "query_results", &[]).await?;
+
+    // Step 9: Verify the results contain the expected data
```

Comment:
See if we can query for the result table ID and do a strict equality check on its contents.